packages/text-split-with-code/index.test.js

```diff
@@ -3,6 +3,7 @@ import TextSplitWithCode from './'
 
 const baseProps = {
   textSplit: {
+    className: 'g-text-split',
```

Comment:
Have updated `text-split-with-*` tests to retain their existing behaviour, by using the newly added `className` prop on `text-split`.